frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <v-skeleton-loader :loading="loading" type="table-tbody">
```

Comment:
can we move v-skeleton-loader to inside v-card outline so this section while loading will look similar to organization info?
![image](https://github.com/user-attachments/assets/0bfc7e02-b001-4c86-89f6-160087112428)

Reply:
I moved skeleton loader out of the component as it wasn't playing nice with the file uploader. It should be good now